multicurrency_journal_entries/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Multicurrency Journal Entries
+=============================
+
+This module was written to extend the functionality of the e-invoicing module
+(account) in order to be able to create journal entries in a currncy different
+than the currency of the company
+
+Installation
+============
+
+To install this module, you need to:
+
+* clone the branch 8.0 of the repository https://github.com/OCA/account-closing
```

Comment:
* clone the branch 8.0 of the repository https://github.com/OCA/account-financial-tools